File: circle.py
import time

N = 10000
start = time.time()

n_points_in_circle = 0
for x in range(-N, N + 1):
    for y in range(-N, N + 1):
        if x ** 2 + y ** 2 < N ** 2:
            n_points_in_circle += 1

duration = time.time() - start

print("%d %fs" % (n_points_in_circle, duration))

File: circle.php
<?php

define('N', 10000);
$start = microtime(true);

$n_points_in_circle = 0;
for ($x = -N; $x <= N + 1; $x++) {
    for ($y = -N; $y <= N + 1; $y++) {
        if ($x * $x + $y * $y < N * N) {
            $n_points_in_circle++;
        }
    }
}

$duration = microtime(true) - $start;

echo "{$n_points_in_circle} {$duration}s";